src/test/java/com/team2/the_shop/stepDef_SmokeTest.java

```diff
@@ -1,4 +1,46 @@
 package com.team2.the_shop;
 
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.junit.jupiter.api.Assertions;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+
 public class stepDef_SmokeTest {
```

Comment:
Du glömde lägga till ditt namn som kommentar på klassen. Se mitt förslag nedan.

```suggestion
public class stepDef_SmokeTest { // Skrivet av Linn Bergström
```